### ISSUE TITLE
Unable to Choose metrics in Model.compile()

### ISSUE THREAD

**Issue by yourtheron** [keras-team/keras#12933]
I use tensorflow backend.

When I want to train a model, I wish to set the metrics as 'accuracy' or 'precision'. yet in Kear's latest doc, I cannot find any information but merely saying, I quote

metrics: List of metrics to be evaluated by the model during training and testing. Typically you will use metrics=['accuracy']. To specify different metrics for different outputs of a multi-output model, you could also pass a dictionary, such as metrics={'output_a': 'accuracy'}.

 https://keras.io/models/sequential/

So I tried both 'acc' and 'accuracy', but the system said there was no such keyword.

I then looked it up in another Keras' doc, http://faroit.com/keras-docs/1.2.2/metrics/
In the list of keywords, I chose 'percision', yet it did not work, either.

So my questions is
1. What should I do so as to choose accuracy or precision for model's compilation?
2. Which docs should I follow in the future? If the former one is newer, should some useful information such as this be included?

Thanks.
